packages/squiggle-lang/src/rescript/Distributions/SampleSetDist/SampleSetDist.res

```diff
@@ -62,7 +78,19 @@ let sampleN = (t: t, n) => {
 }
 
 //TODO: Figure out what to do if distributions are different lengths. ``zip`` is kind of inelegant for this.
-let map2 = (~fn: (float, float) => float, ~t1: t, ~t2: t) => {
+let map2 = (
```

Comment:
Any reason this isn't in `E.R`?

Reply:
No, this is over SampleSetDists?

Reply:
This is specifically for the SampleSetDist type. We could move it to array, but we'd still want this function. I wouldn't worry about it for this PR.